### PR TITLE
[TDV-686] Feature/block banner

### PR DESCRIPTION
**Pull request by Datikzz**

### CODE REVIEW

File: src/vue/App.vue [hunk 1]
```diff
@@ -6,6 +6,12 @@
     />
 
     <template v-if="isLoggedIn && isNavigationRendered">
+      <warning-banner
+        v-if="isAccountBlocked"
+        :message-id="'warning-banner.blocked-desc'"
+        :message-type="'danger'"
```
Comment (max-nes) [inline src/vue/App.vue]: `:message-type="'danger'"` => `message-type="danger"`

File: CHANGELOG.md [hunk 2]
```diff
@@ -16,6 +16,7 @@ for further information about branching and tagging conventions.
 ## [1.6.0-x.0] - 2019-04-24
 #### Added
 - Copy button next to email values
+- Show message for blocked user
```
Comment (max-nes) [inline CHANGELOG.md]: move to unreleased

File: src/vue/common/WarningBanner.vue [hunk 3]
```diff
@@ -19,19 +22,27 @@ export default {
...
+  &--danger {
+    background-color: $col-danger;
+  }
 }
 
 .warning-banner__message-link {
   color: #fff;
   cursor: pointer;
   text-decoration: underline;
+  text-align: center;
```
Comment (max-nes) [inline src/vue/common/WarningBanner.vue]: `.warning-banner__message-link` is unused isnt it?

Comment (Datikzz) [reply src/vue/common/WarningBanner.vue]: yap, removed

File: CHANGELOG.md (hunk 2 above)
Comment (Datikzz) [reply CHANGELOG.md]: done

File: src/vue/App.vue (hunk 1 above)
Comment (Datikzz) [reply src/vue/App.vue]: done

**Pull request merged**